src/api/update-volunteer.ts

```
@@ -25,9 +25,8 @@ const updateVolunteer = async (
         ...object,
         [key]: remove ? '' : uid,
     };
-    return await updateDoc(docRef, object)
-        .then((response) => response)
-        .catch((response) => response);
+
+    return updateDoc(docRef, object);
```

This is only used in the `VolunteerForm` component, and it's being called from a `useMutation`, which can handle errors properly, so I've changed this function to not do silent error returning anymore.